loopchain/channel/channel_service.py

```diff
@@ -22,17 +22,18 @@
 
 from loopchain import configure as conf
 from loopchain import utils
-from loopchain.baseservice import (BroadcastScheduler, BroadcastSchedulerFactory, ObjectManager, CommonSubprocess,
+from loopchain.baseservice import (BroadcastScheduler, BroadcastSchedulerFactory, ObjectManager,
+                                   CommonSubprocess,
```

Comment:
Any reason to break line for `CommonSubprocess`?

Reply:
Haha my editor hard wrap is 110. It allows editing three pages on one screen.